lint/rule_panel_units.go

```diff
@@ -80,10 +80,7 @@ func NewPanelUnitsRule() *PanelRuleFunc {
 				if p.Type == "stat" {
 					var opts StatOptions
 					err := json.Unmarshal(p.Options, &opts)
-					if err != nil {
-						r.AddError(d, p, err.Error())
-					}
-					if hasReduceOptionsNonNumericFields(&opts.ReduceOptions) {
```

Comment:
Had to go to the whole file to grok the change here, I was worried you were skipping the error case ;}

Nice simplification, thank you!